src/lib/openables.ts

```diff
@@ -234,7 +250,21 @@ const osjsOpenables: UnifiedOpenable[] = [
 			'larrans small chest',
 			"larran's small chest"
 		],
-		output: Openables.LarransChest.table,
+		output: async (
+			args: OpenArgs
+		): Promise<{
+			bank: Bank;
+		}> => {
+			const chest = new LarransChestOpenable(LarransChest);
+			const fishLvl = args.user.skillLevel(SkillsEnum.Fishing);
+			const larransOptions: any = {
```

Comment:
any here too

Reply:
This one errors out without the any. I've tried a few things but couldn't get it to work without the any

Reply:
nvm i got it working.